frontend/src/services/GraphDataSource.ts

```diff
@@ -293,10 +302,11 @@ export class GraphDataSource {
     }
 
     this.emit('loadStart', isPreviousDataInvalid, fetchParams);
+
     if (fetchParams.node) {
-      this.fetchDataForNode(restParams);
+      this.fetchDataForNode(restParams, cluster);
     } else {
-      this.fetchDataForNamespaces(restParams);
+      this.fetchDataForNamespaces(restParams, cluster);
```

Comment:
This should never take `cluster` see my comment above about removing this public signature.